packages/core/test/runScenarios.spec.js

```diff
@@ -1,21 +1,22 @@
 const test = require('ava')
-const { loadScenarios } = require('./scenarios/index')
+const { loadScenarios } = require('./scenarios')
 const { runScenario, runAndTestScenario } = require('./util')
 
-test('Run scenarios', async (t) => {
+;(async () => {
   for await (const scenario of loadScenarios()) {
-    t.log(`Running Core Scenario: ${scenario.name}`)
-    await runAndTestScenario(t, scenario, ({ scenario }) =>
-      runScenario({ scenario })
-    )
-  }
-})
+    test.serial(`core scenario: ${scenario.name}`, async (t) => {
+      await runAndTestScenario(t, scenario, ({ scenario }) =>
+        runScenario({ scenario })
+      )
+    })
 
-test('Run scenarios with precompiled intializer', async (t) => {
-  for await (const scenario of loadScenarios()) {
-    t.log(`Running Core Scenario: ${scenario.name}`)
-    await runAndTestScenario(t, scenario, ({ scenario }) =>
-      runScenario({ scenario, runWithPrecompiledModules: true })
+    test.serial(
+      `core scenario (precompiled initializer): ${scenario.name}`,
+      async (t) => {
+        await runAndTestScenario(t, scenario, ({ scenario }) =>
+          runScenario({ scenario, runWithPrecompiledModules: true })
+        )
+      }
     )
   }
-})
+})()
```

Comment:
Should there be a `.catch` for logging errors thrown in tests?

Reply:
I don't think it's necessary.  AVA traps all the test errors, and if we fail somewhere in the loop, then popping an unhandled rejection is desirable.